File: jm-redis-cacheable/src/test/java/com/example/rediscacheable/RedisCacheableApplicationTests.java
package com.example.rediscacheable;

import com.example.rediscacheable.server.domain.User;
import com.example.rediscacheable.server.service.impl.UserServiceImpl;
import org.junit.jupiter.api.Test;
import org.springframework.boot.test.context.SpringBootTest;

import javax.annotation.Resource;

@SpringBootTest
class RedisCacheableApplicationTests {

    @Resource
    UserServiceImpl userService;

    @Test
    void contextLoads() {
        System.out.println(userService.getUser(1L));
    }

    @Test
    void contextLoads1() {
        User user = new User();
        user.setId(1L);
        user.setUsername("admin");
        user.setPassword("123456");
        System.out.println(userService.updateUser(user));
    }

}

File: kg-redis-cacheable/src/test/kotlin/com/example/rediscacheable/RedisCacheableApplicationTests.kt
package com.example.rediscacheable

import com.example.jgrediscacheable.serve.service.UserService
import org.junit.jupiter.api.Test
import org.springframework.boot.test.context.SpringBootTest
import javax.annotation.Resource

@SpringBootTest
class RedisCacheableApplicationTests {

    @Resource
    private lateinit var userService: UserService;
    @Test
    fun contextLoads() {
        println(userService.getUser(1L))
    }
}
